webpack.common.js

```diff
@@ -86,37 +90,37 @@ module.exports = (env = {}) => ({
         test: /\.jsx?$/,
         exclude: /node_modules/,
         use: [
-          'babel-loader',
-          {
-            loader: 'eslint-loader',
-            options: { fix: true }
-          }
+          'babel-loader'
         ]
       },
       {
         test: /\.css$/,
         use: [
-          MiniCssExtractPlugin.loader,
+          isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
```

Comment:
i think this belongs to dev config instead, since we already have one

Reply:
Then we have to duplicate those `module.rules` in both `webpack.dev.js` and `webpack.prod.js`, that would be much difficult to maintain than current one in my opinion.

Reply:
yeah it looks like there is no good way to do it besides iterating and modifying the array, keep it here